testing/btest/cluster/websocket/terminate-while-queuing.zeek

```diff
@@ -0,0 +1,94 @@
+# @TEST-DOC: Regression test for #4420. Clients publish fast and Zeek terminates after receiving 1000 events. Previously this would result in a hang at Zeek shutdown.
+#
+# @TEST-REQUIRES: python3 -c 'import websockets.sync'
+#
+# @TEST-PORT: WEBSOCKET_PORT
+#
+# @TEST-EXEC: cp $FILES/ws/wstest.py .
+# @TEST-EXEC: zeek -b --parse-only manager.zeek
+# @TEST-EXEC: python3 -m py_compile client.py
+#
+# @TEST-EXEC: btest-bg-run manager "ZEEKPATH=$ZEEKPATH:.. && zeek -b ../manager.zeek >out"
+# @TEST-EXEC: btest-bg-run client "python3 ../client.py >out"
+#
+# @TEST-EXEC: btest-bg-wait 30
+# @TEST-EXEC: sort ./manager/out > ./manager/out.sorted
+# @TEST-EXEC: btest-diff ./manager/out.sorted
+# @TEST-EXEC: btest-diff ./manager/.stderr
+# @TEST-EXEC: btest-diff ./client/out
+# @TEST-EXEC: btest-diff ./client/.stderr
+
+# # @TEST-START-FILE manager.zeek
+redef exit_only_after_terminate = T;
+
+# Force dispatcher queue being full quickly!
+redef Cluster::default_websocket_max_event_queue_size = 1;
+
+global ping_count = 0;
+global ping: event(msg: string, c: count) &is_used;
+
+global clients: set[string] = set();
+
+event ping(client: string, n: count) &is_used
+	{
+        ++ping_count;
+        add clients[client];
+
+        if ( ping_count == 1000 )
```

Comment:
Is this `Cluster::Backend::ZeroMQ::log_rcvhwm` so we could get rid of the magic var? An `assert` here would make it clear for the Python code below where the value comes from.

Reply:
It's an arbitrary value that's large enough so the test takes some time and the clients can build up queued events. I'll leave as is.